aacgain: Add package.

robbinch opened the pull request:
This adds aacgain which is a normalizer for AAC (.m4a) audio files.

robbinch committed:
aacgain: Add package.

This adds aacgain which is a normalizer for AAC (.m4a) audio files.

pSub commented on pkgs/applications/audio/aacgain/default.nix:
```diff
@@ -0,0 +1,72 @@
+stdenv.mkDerivation {
+  name = "aacgain-1.9";
+  srcs = [
+    (fetchgit {
```
Please use fetchFromGitHub, with the release tag as `rev`, that avoids cloning the full repository.

pSub commented on pkgs/applications/audio/aacgain/default.nix:
```diff
@@ -0,0 +1,72 @@
+
+  buildPhase = ''
+    cd $sourceRoot/mp4v2
+    make libmp4v2.la
```
Do you really need to build those dependencies here?

robbinch committed:
aacgain: Use fetchFromGitHub instead of fetchgit.

robbinch commented:
Fixed to use fetchFromGitHub.

@pSub Unfortunately, yes. The steps above follows upstream instructions on building for Linux, which is added as an afterthought to a Windows program, therefore building it is a bit unconventional.

robbinch committed:
aacgain: Use more up-to-date upstream repository.

Use more up-to-date upstream repostiory, which allows for a cleaner
build. It includes mp4v2 and faad, which will be built statically and
linked to the resulting binary.

garbas commented:
merged [here](http://git.io/vI5av)
